tests/notification/README.md

```diff
@@ -0,0 +1,33 @@
+# Autopush Notification Integration Tests
```

Comment:
nit: perhaps this content should be in [testing.md](https://github.com/mozilla-services/autopush-rs/blob/master/docs/src/testing.md)

cc: @taddes